src/plugins/kibana_react/common/eui_provider.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { DecoratorFn } from '@storybook/react';
```

Comment:
Another argument to have this in `kbn-storybook`, as this would introduce a dependency to `kibana_react` that's not used in any other context.

Reply:
It is used in `src/plugins/kibana_react/common/eui_styled_components.tsx` (which is why I put it in kibana_react) but scoping to kbn-storybook makes more sense to me.